src/hooks/compose.ts

```diff
@@ -0,0 +1,23 @@
+import { Hooks } from '../types'
```

Comment:
Adds the ability to "compose" multiple hooks, [as seen here](https://github.com/wagmi-dev/create-wagmi/pull/20/files#diff-2b2c551d87ab178c988bb32e57740c92b86dcb3a7173d739e4631af108c8fae5R12-R42).